### PR TITLE
MAUI-860107: UG content for new shapes in ImageEditor

### PR DESCRIPTION
**Pull request by Shalini-Ashokan**
Hi @karthikraja-arumugam 

 I have prepared UG content for new shapes in Maui image editor control. Could you please review it and let me know if you have any concerns?

Regards,
Shalini A.

### CODE REVIEW

File: MAUI/ImageEditor/shape.md [hunk 1]
```diff
@@ -63,6 +104,7 @@ Customize the appearance of each shape using the [`ImageEditorShapeSettings`](ht
...
 * [`StrokeThickness`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.ImageEditor.ImageEditorShapeSettings.html#Syncfusion_Maui_ImageEditor_ImageEditorShapeSettings_StrokeThickness) - Specifies the stroke width of the shapes. It is not applicable for [`AnnotationShape.Rectangle`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.ImageEditor.AnnotationShape.html#Syncfusion_Maui_ImageEditor_AnnotationShape_Rectangle) and [`AnnotationShape.Circle`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.ImageEditor.AnnotationShape.html#Syncfusion_Maui_ImageEditor_AnnotationShape_Circle) in the filled state.
 * [`Opacity`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.ImageEditor.ImageEditorAnnotationSettings.html#Syncfusion_Maui_ImageEditor_ImageEditorAnnotationSettings_Opacity): Specifies the opacity of shape annotation. This value ranges from 0 to 1.
+* `Points`: Specifies the vertex points of the shapes. Position the shapes wherever you want on the image.
```
Comment (karthikraja-arumugam) [inline MAUI/ImageEditor/shape.md:107]: mention this is applicable for polygon and polyline

File: MAUI/ImageEditor/shape.md [hunk 2]
```diff
@@ -51,6 +51,47 @@ The [`AnnotationShape`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Imag
 * DoubleArrow
 * DottedArrow
 * DottedDoubleArrow
+* Polygon
+* Polyline
+
+## Path annotation
```
Comment (karthikraja-arumugam) [inline MAUI/ImageEditor/shape.md:57]: Do not mention as path annotation, add separate topics, polygon and polyline

File: MAUI/ImageEditor/shape.md (hunk 1 above)
Comment (Shalini-Ashokan) [reply MAUI/ImageEditor/shape.md]: Mentioned

File: MAUI/ImageEditor/shape.md (hunk 2 above)
Comment (Shalini-Ashokan) [reply MAUI/ImageEditor/shape.md]: Added separate topics for polygon and polyline

File: MAUI/ImageEditor/shape.md [hunk 3]
```diff
@@ -51,6 +51,84 @@ The [`AnnotationShape`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Imag
 * DoubleArrow
 * DottedArrow
 * DottedDoubleArrow
+* Polygon
+* Polyline
+
+## Polygon
```
Comment (karthikraja-arumugam) [inline MAUI/ImageEditor/shape.md:57]: Images are not clear, add shapes without background image and try to add large size shapes. For polyline add shape like in whatsnew image

File: MAUI/ImageEditor/shape.md [hunk 4]
```diff
@@ -63,6 +141,7 @@ Customize the appearance of each shape using the [`ImageEditorShapeSettings`](ht
...
 * [`StrokeThickness`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.ImageEditor.ImageEditorShapeSettings.html#Syncfusion_Maui_ImageEditor_ImageEditorShapeSettings_StrokeThickness) - Specifies the stroke width of the shapes. It is not applicable for [`AnnotationShape.Rectangle`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.ImageEditor.AnnotationShape.html#Syncfusion_Maui_ImageEditor_AnnotationShape_Rectangle) and [`AnnotationShape.Circle`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.ImageEditor.AnnotationShape.html#Syncfusion_Maui_ImageEditor_AnnotationShape_Circle) in the filled state.
 * [`Opacity`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.ImageEditor.ImageEditorAnnotationSettings.html#Syncfusion_Maui_ImageEditor_ImageEditorAnnotationSettings_Opacity): Specifies the opacity of shape annotation. This value ranges from 0 to 1.
+* `Points`: Specifies the vertex points of the shapes. Position the shapes wherever you want on the image. It is applicable for polygon and polyline.
```
Comment (karthikraja-arumugam) [inline MAUI/ImageEditor/shape.md:144]: `Points`: Specifies the coordinates of the vertices that make up the shape. Arrange the shapes at your desired location within the image. This attribute is applicable for only `Polygon` and `Polyline` shape types.

File: MAUI/ImageEditor/shape.md [hunk 5]
```diff
@@ -51,6 +51,84 @@ The [`AnnotationShape`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Imag
...
+    }
+
+{% endhighlight %}
+{% endtabs %}
+
+![Polygon annotation in .NET Maui ImageEditor](images/shape/imageeditor-polygon-annotation.png)
+
+## Polyline
+
+It draws a series of connected straight lines and used to create triangles, pentagons, and other shapes with multiple points.
```
Comment (karthikraja-arumugam) [inline MAUI/ImageEditor/shape.md:96]: It draws a series of connected straight lines. It is similar to a polygon, except the last point in a polyline is not connected to the first point.

File: MAUI/ImageEditor/shape.md [hunk 6]
```diff
@@ -63,6 +141,7 @@ Customize the appearance of each shape using the [`ImageEditorShapeSettings`](ht
 * [`IsFilled`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.ImageEditor.ImageEditorShapeSettings.html#Syncfusion_Maui_ImageEditor_ImageEditorShapeSettings_IsFilled) - Enables or disables the fill color of the shapes. This is applicable only for [`AnnotationShape.Rectangle`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.ImageEditor.AnnotationShape.html#Syncfusion_Maui_ImageEditor_AnnotationShape_Rectangle) and [`AnnotationShape.Circle`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.ImageEditor.AnnotationShape.html#Syncfusion_Maui_ImageEditor_AnnotationShape_Circle) shape types.
```
Comment (karthikraja-arumugam) [inline MAUI/ImageEditor/shape.md:141]: Mention isfilled applicable for Polygon

File: MAUI/ImageEditor/shape.md [hunk 7]
```diff
@@ -63,6 +141,7 @@ Customize the appearance of each shape using the [`ImageEditorShapeSettings`](ht
...
 * [`StrokeThickness`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.ImageEditor.ImageEditorShapeSettings.html#Syncfusion_Maui_ImageEditor_ImageEditorShapeSettings_StrokeThickness) - Specifies the stroke width of the shapes. It is not applicable for [`AnnotationShape.Rectangle`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.ImageEditor.AnnotationShape.html#Syncfusion_Maui_ImageEditor_AnnotationShape_Rectangle) and [`AnnotationShape.Circle`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.ImageEditor.AnnotationShape.html#Syncfusion_Maui_ImageEditor_AnnotationShape_Circle) in the filled state.
```
Comment (karthikraja-arumugam) [inline MAUI/ImageEditor/shape.md:142]: mention not applicable for polygon in filled state

File: MAUI/ImageEditor/shape.md [hunk 8]
```diff
@@ -51,6 +51,91 @@ The [`AnnotationShape`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Imag
 * DoubleArrow
 * DottedArrow
 * DottedDoubleArrow
+* Polygon
+* Polyline
+
+## Polygon
+
+A polygon is formed by connecting a series of straight lines, and in addition to the `Points` collection includes specific points, a line is automatically connects the first and last points.
```
Comment (florenceliliana) [inline MAUI/ImageEditor/shape.md:59]: A polygon is formed by connecting a series of straight lines, and in addition to the `Points` collection including specific points, a line automatically connects the first and last points.

File: MAUI/ImageEditor/shape.md [hunk 9]
```diff
@@ -60,9 +145,10 @@ Customize the appearance of each shape using the [`ImageEditorShapeSettings`](ht
...
+* [`IsFilled`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.ImageEditor.ImageEditorShapeSettings.html#Syncfusion_Maui_ImageEditor_ImageEditorShapeSettings_IsFilled) - Enables or disables the fill color of the shapes. This is applicable only for [`AnnotationShape.Rectangle`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.ImageEditor.AnnotationShape.html#Syncfusion_Maui_ImageEditor_AnnotationShape_Rectangle), [`AnnotationShape.Circle`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.ImageEditor.AnnotationShape.html#Syncfusion_Maui_ImageEditor_AnnotationShape_Circle) and `AnnotationShape.Polygon` shape types.
```
Comment (florenceliliana) [inline MAUI/ImageEditor/shape.md:148]: * [`IsFilled`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.ImageEditor.ImageEditorShapeSettings.html#Syncfusion_Maui_ImageEditor_ImageEditorShapeSettings_IsFilled) - Enables or disables the fill color of the shapes. This is applicable only for [`AnnotationShape.Rectangle`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.ImageEditor.AnnotationShape.html#Syncfusion_Maui_ImageEditor_AnnotationShape_Rectangle), [`AnnotationShape.Circle`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.ImageEditor.AnnotationShape.html#Syncfusion_Maui_ImageEditor_AnnotationShape_Circle), and `AnnotationShape.Polygon` shape types.

File: MAUI/ImageEditor/shape.md [hunk 10]
```diff
@@ -60,9 +145,10 @@ Customize the appearance of each shape using the [`ImageEditorShapeSettings`](ht
...
+* [`StrokeThickness`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.ImageEditor.ImageEditorShapeSettings.html#Syncfusion_Maui_ImageEditor_ImageEditorShapeSettings_StrokeThickness) - Specifies the stroke width of the shapes. It is not applicable for [`AnnotationShape.Rectangle`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.ImageEditor.AnnotationShape.html#Syncfusion_Maui_ImageEditor_AnnotationShape_Rectangle), [`AnnotationShape.Circle`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.ImageEditor.AnnotationShape.html#Syncfusion_Maui_ImageEditor_AnnotationShape_Circle) and `AnnotationShape.Polygon` in the filled state.
```
Comment (florenceliliana) [inline MAUI/ImageEditor/shape.md:149]: * [`StrokeThickness`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.ImageEditor.ImageEditorShapeSettings.html#Syncfusion_Maui_ImageEditor_ImageEditorShapeSettings_StrokeThickness) - Specifies the stroke width of the shapes. It is not applicable for [`AnnotationShape.Rectangle`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.ImageEditor.AnnotationShape.html#Syncfusion_Maui_ImageEditor_AnnotationShape_Rectangle), [`AnnotationShape.Circle`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.ImageEditor.AnnotationShape.html#Syncfusion_Maui_ImageEditor_AnnotationShape_Circle), and `AnnotationShape.Polygon` in the filled state.

File: MAUI/ImageEditor/shape.md [hunk 11]
```diff
@@ -60,9 +145,10 @@ Customize the appearance of each shape using the [`ImageEditorShapeSettings`](ht
...
 * [`Opacity`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.ImageEditor.ImageEditorAnnotationSettings.html#Syncfusion_Maui_ImageEditor_ImageEditorAnnotationSettings_Opacity): Specifies the opacity of shape annotation. This value ranges from 0 to 1.
+* `Points`: Specifies the coordinates of the vertices that make up the shape. Arrange the shapes at your desired location within the image. This attribute is applicable for only `Polygon` and `Polyline` shape types.
```
Comment (florenceliliana) [inline MAUI/ImageEditor/shape.md:151]: * `Points`: Specifies the coordinates of the vertices that make up the shape. Arrange the shapes at your desired location within the image. This attribute applies to only `Polygon` and `Polyline` shape types.

**Review by florenceliliana — CHANGES_REQUESTED** [review]
The content review is completed.

File: MAUI/ImageEditor/shape.md [hunk 12]
```diff
@@ -51,6 +51,91 @@ The [`AnnotationShape`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Imag
...
+                },
+            });
+    }
+
+{% endhighlight %}
+{% endtabs %}
+
+![Polygon annotation in .NET Maui ImageEditor](images/shape/imageeditor-polygon-annotation.png)
+
+## Polyline
```
[the remark this answers is not in the document]
Comment (karthikraja-arumugam) [reply MAUI/ImageEditor/shape.md:97]: still image not replaced, polyline should be withnn the screen

**Review by karthikraja-arumugam — CHANGES_REQUESTED** [review]

**Review by Jeyasri-Murugan — APPROVED** [review]

**Review by karthikraja-arumugam — APPROVED** [review]

**Pull request merged**